ptx/S_12_3_ParamLineIntegrals.ptx

```diff
@@ -2,427 +2,264 @@
       <me>
-        \int_C\vF\cdot d\vr = \int_C F_1\, dx + F_2\, dy + F_3\, dz
+        \int_C\vF\cdot d\vr = \int_C \langle F_1, F_2, F_3 \rangle \cdot \langle dx, dy, dz \rangle= \int_C F_1\, dx + F_2\, dy + F_3\, dz
       </me>.
+      A line integral in the form of <m>\int_C F_1\, dx + F_2\, dy + F_3\, dz</m> is called the <term><idx>differential form of a line integral</idx></term>.
     </p>
 
     <p>
-      (If <m>\vF</m> is a vector field in <m>\R^2</m>,
-      the <m>F_3\, dz</m> term is
-      omitted.) As a concrete example,
-      if <m>\vF(x,y,z) = \langle
-      x^2y,z^3,x\cos(z)\rangle</m> and <m>C</m> is some oriented curve in <m>\R^3</m>,
-      then
+      (If <m>\vF</m> is a vector field in <m>\R^2</m>, the <m>F_3\, dz</m> term is omitted.) As a concrete example, if <m>\vF(x,y,z) = \langle x^2y,z^3,x\cos(z)\rangle</m> and <m>C</m> is some oriented curve in <m>\R^3</m>, then
       <me>
         \int_C\vF\cdot d\vr = \int_C x^2y\, dx + z^3\, dy + x\cos(z)\, dz
       </me>.
     </p>
 
     <p>
-      It is important to recognize that the integral on the right-hand side
-      is still a line integral and must be evaluated using techniques for
-      evaluating line integrals.
-      We cannot simply try to treat it as if it
-      were a definite integral of a function of one variable.
+      It is important to recognize that the integral on the right-hand side is still a line integral and must be evaluated using techniques for evaluating line integrals. We cannot simply try to treat it as if it were a definite integral of a function of one variable.
     </p>
   </subsection>
 
   <subsection>
-    <title>Independence of Parameterization</title>
+    <title>Independence of Parameterization for a Fixed Curve</title>
     <p>
-      Up to this point, we've just been choosing whatever parameterization
-      of an oriented curve <m>C</m> came to mind,
-      and our argument for how we can
-      use parameterizations to calculate line integrals did not depend on
-      the specific choice of parameterization.
-      However, it is not
-      immediately obvious that different parameterizations don't result in
-      different values of the line integral.
-      Our next example explores this question.
+      Up to this point, we've just been choosing whatever parameterization of an oriented curve <m>C</m> came to mind, and our argument for how we can use parameterizations to calculate line integrals did not depend on the specific choice of parameterization. However, it is not immediately obvious that different parameterizations don't result in different values of the line integral. Our next example explores this question.
     </p>
 
     <example xml:id="ex_12_3_param-indep">
       <statement>
```

Comment:
When an example is stated as "Do ___", then `statement` and `solution` are appropriate. If the example is more in narrative form without a clear problem statement, I think the best practice is to omit the `statement`. I have rephrased at least one example so that using `statement` made sense. Feel free to do likewise.